test/test.js

```diff
@@ -21,19 +21,20 @@ versions.forEach((version) => describe(`Chunk implementation for minecraft ${ver
 
   const isPostFlattening = version.startsWith('1.13') || version.startsWith('1.14') ||
```

Comment:
These test cases should probably be using mc-data features

Reply:
Indeed